### PR TITLE
Document new "Epona Push" bug

### PR DESCRIPTION
**Pull request by fig02**
a lengthy video on the subject for those interested: https://www.youtube.com/watch?v=VAqyqfgxnSk
(decomp contributors who decide to watch can probably skip the collision_check primer lol)

### CODE REVIEW

**Review by Dragorn421 — COMMENTED** [review]
I'm not sure about copypasting the same bug comment over several files, I would rather have it written out in full only once and have other spots reference it like
```
//! @bug See bug description at the end of `EnHorse_SkinCallback1`
```

File: src/overlays/actors/ovl_En_Horse/z_en_horse.c [hunk 1]
```diff
@@ -3783,6 +3783,10 @@ void EnHorse_SkinCallback1(Actor* thisx, GlobalContext* globalCtx, PSkinAwb* ski
             this->jntSph.elements[i].dim.modelSphere.radius * this->jntSph.elements[i].dim.scale;
     }
 
+    //! @bug Setting OC and AC in a draw function allows for duplicate entries to be added to their respective lists
+    //! under certain conditions, like when pausing and unpausing the game. Actors will draw but not update in this case, so
```
[the remark this answers is not in the document]
Comment (fig02) [reply src/overlays/actors/ovl_En_Horse/z_en_horse.c:3787]: Yeah you are correct that ClearContext is what would actually clear it. Im more so using "actors updating" as a high level description for this relevant block of gameplay_update. Cause update does run, but this important section gets blocked pretty early on by a pause state check

**Comment by fig02** [comment]
I personally dont have an issue with copying the comment where relevant

File: src/overlays/actors/ovl_En_Horse/z_en_horse.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (Dragorn421) [reply src/overlays/actors/ovl_En_Horse/z_en_horse.c:3787]:
```suggestion
    //! under certain conditions, like when pausing and unpausing the game. Actors will draw, but most update logic will not run in this case, so
```
maybe?

**Comment by Roman971** [comment]
I'm leaning on the opinion that we shouldn't copy the bug comment in multiple places, because all relevant parts of code are extremely similar and we might need to change the comment in the future, so it would keep things simpler to have it written out in one place.

**Comment by fig02** [comment]
Changed it for now to reduce dupes, but still not a fan of it. When I see a bug tag I wanna be told what the problem is asap and not have to go play hide and seek to find it lol
But I do recognize that the comment wording may need to change, especially if/when things like the collider state names change

**Review by Dragorn421 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Pull request merged**